examples/default.yaml

```diff
@@ -297,6 +297,9 @@ audio:
   # QEMU audiodev, e.g., "none", "coreaudio", "pa", "alsa", "oss".
   # VZ driver, use "vz" as device name
   # Choosing "none" will mute the audio output, and not play any sound.
+  # Choosing "default" will pick a suitable of: coreudio, pa, dsound, oss.
+  # As of QEMU v6.2 the default is to create a disconnected sound device
```

Comment:
This is still ambiguous if "the default" means when it is set to `"default"` or to "the builtin default" `""`.

But maybe it doesn't really matter, practically speaking.